capa-controller-role/policies/crossplane-policy.json

```diff
@@ -14,7 +14,18 @@
                 "ec2:RevokeSecurityGroupIngress",
                 "cloudwatch:*",
                 "sqs:*",
-                "events:*"
+                "events:*",
+                "iam:GetRole",
+                "iam:CreateRole",
+                "iam:DeleteRole",
+                "iam:TagRole",
```

Comment:
maybe also `iam:TagPolicy` and `iam:UntagRole` and `iam:UntagPolicy`

Reply:
done